tests/into.rs

```diff
@@ -1043,3 +1043,366 @@ mod multi_field {
         }
     }
 }
+
+mod field_attr {
+    use super::*;
+
+    mod fields_only {
+        use super::*;
+
+        #[derive(Debug, Into, Clone, Copy)]
+        struct Tuple(#[into] i32, f64, #[into] f64);
+
+        impl From<Tuple> for (i32, f64, f64) {
+            fn from(value: Tuple) -> Self {
+                (value.0, value.1, value.2)
+            }
+        }
+
+        impl From<Tuple> for (i32, f64) {
+            fn from(value: Tuple) -> Self {
+                (value.0, value.2)
+            }
+        }
```

Comment:
Why are these impls here?

Reply:
The idea was to test that no extra impls get generated.

For instance, no impl should be generated using all the fields if there's at least one field attribute, and if there are types specified in an attribute the field's type shouldn't  get one either.